polkadot/cli/src/cli.rs

```diff
@@ -159,6 +159,13 @@ pub struct RunCmd {
 	/// networks.
 	#[arg(long)]
 	pub keep_finalized_for: Option<u32>,
+
+	/// Overrides `HOLD_OFF_DURATION` in collator_protocol/validator_side. The value is in
+	/// milliseconds.
+	///
+	///  **Dangerous!** Do not touch unless explicitly advised to.
+	#[arg(long, hide = true)]
+	pub collator_protocol_hold_off: Option<u64>,
```

Comment:
nit: `keep_finalized_for` uses `u32` here `u64`, is that intentional. I'd prefer a `pub type DurationMs = u64` or similar instead of primitives like u32/u64 for time, using the type to convey the unit of time.

Reply:
[from_millis](https://doc.rust-lang.org/std/time/struct.Duration.html#method.from_millis) uses `u64` as an input, that's why I've picked `u64` here. 
I haven't used `Duration` or something similar here because as far as I understand clap expects simple types as they get parsed from the command line.